### PR TITLE
Collapsible filter panel with column layout

### PR DESCRIPTION
**Pull request by ritters**
@periklis @mmoll @calbrecht @elger @xalopp @andythedandy @danielhallmann please review.

### CODE REVIEW

File: packages/mf-grid-filter-form/examples/simple-grid/app/view/main/Main.js [hunk 1]
```diff
@@ -58,12 +58,13 @@ Ext.define('SimpleGrid.view.main.Main', {
                 }]
             }]
         }, {
-            title: 'Grid with filter from with paging bar',
+            title: 'Grid with collapsible filter from with paging bar',
```
Comment (mmoll) [inline packages/mf-grid-filter-form/examples/simple-grid/app/view/main/Main.js]: from -> form

File: packages/mf-grid-filter-form/examples/simple-grid/app/view/main/Main.js [hunk 2]
```diff
@@ -85,12 +86,14 @@ Ext.define('SimpleGrid.view.main.Main', {
                 height: 300
             }]
         }, {
-            title: 'Grid with ordered paging bar',
+            title: 'Grid with ordered paging bar and 2 columns form',
```
Comment (mmoll) [inline packages/mf-grid-filter-form/examples/simple-grid/app/view/main/Main.js]: evtl. form -> filter form

File: packages/mf-grid-filter-form/src/grid/feature/FilterForm.js [hunk 3]
```diff
@@ -218,5 +283,111 @@ Ext.define('Mayflower.grid.feature.FilterForm', {
...
+            displayedFields;
+
+        formColumnItems = [];
+        displayedFields = me.getDisplayedFilterElements(filterItems);
+        hiddenFields = me.getHiddenFilterElements(filterItems);
+        elements = displayedFields.length;
+        columnElementsNumber = parseInt(elements / me.columns, 10);
+        columnElements = [];
+
+        for (var i = 1; i <= displayedFields.length; i ++) {
```
Comment (mmoll) [inline packages/mf-grid-filter-form/src/grid/feature/FilterForm.js]: It would be nice to have proper variable names instead of i and k!

**Comment by mmoll** [comment]
This is looking good overall. When the final commits from @andythedandy are in, it should be squashed together and GH-10 and GH-12 referenced with "closes" in the commit message.

**Comment by mmoll** [comment]
BTW, we should set the version to 1.1.0, according to semver, as features where added and with this we can reference it in the actual consuming application.

**Comment by mmoll** [comment]
I think we're good to go now, please review again.

**Comment by calbrecht** [comment]
:+1:

**Comment by mmoll** [comment]
thanks all, merging :bomb: :boom:

**Pull request merged**